src/Analyzer/Passes/QueryAnalysisPass.cpp

```diff
@@ -66,6 +66,15 @@
 #include <Analyzer/UnionNode.h>
 #include <Analyzer/InDepthQueryTreeVisitor.h>
 #include <Analyzer/QueryTreeBuilder.h>
+#include <Analyzer/IQueryTreeNode.h>
+#include <Common/ProfileEvents.h>
```

Comment:
We include files from `Common` at the top of the file.

Reply:
addressed it